insights/specs/datasources/ls.py

```diff
@@ -52,7 +54,13 @@ def list_with_la_filtered(broker):
 
 @datasource(HostContext)
 def list_with_lan(broker):
-    return _list_items(Specs.ls_lan_dirs)
+    filters = _list_items(Specs.ls_lan_dirs)
+    if 'specs.fstab_mounted' in filters and FSTab in broker:
```

Comment:
As per the Jira, for now, this filter is only for the SAP rule, I think we could add the `sap_hana_instance` as one dependency here.
```python
from insights.specs.datasources.sap import LocalSpecs as SAPSpecs

if 'specs.fstab_mounted' in filters and SAPSpecs.sap_hana_instance in broker and FSTab in broker:
```